src/frontend/context/ConfigContext.tsx

```diff
@@ -13,67 +12,67 @@ import type { Status } from "../types";
 
 const log = debug("mapeo:ConfigContext");
 
-export type Preset = {|
-  id: string,
-  icon?: string,
-  fields?: string[],
-  geometry: Array<"point" | "area" | "line" | "vertex" | "relation">,
-  terms?: string[],
-  tags: { [string]: any },
-  name: string,
-  sort?: number,
-  matchScore?: number,
-  searchable?: boolean,
-|};
+export type Preset = {
+  id: string;
+  icon?: string;
+  fields?: string[];
+  geometry: Array<"point" | "area" | "line" | "vertex" | "relation">;
+  terms?: string[];
+  tags: { [x: string]: any };
+  name: string;
+  sort?: number;
+  matchScore?: number;
+  searchable?: boolean;
+};
 
 export type {
   TextField,
   SelectOneField,
-  SelectMultipleField,
+  SelectMultipleField, //@ts-ignore
 } from "mapeo-schema";
 
 export type Field = TextField | SelectOneField | SelectMultipleField;
 
-export type PresetWithFields = {|
-  id: string,
-  icon?: string,
-  fields: Field[],
-  geometry: Array<"point" | "area" | "line" | "vertex" | "relation">,
-  terms?: string[],
-  tags: { [string]: any },
-  name: string,
-  sort?: number,
-  matchScore?: number,
-  searchable?: boolean,
-|};
+export type PresetWithFields = {
+  id: string;
+  icon?: string;
+  fields: Field[];
+  geometry: Array<"point" | "area" | "line" | "vertex" | "relation">;
+  terms?: string[];
+  tags: { [x: string]: any };
+  name: string;
+  sort?: number;
+  matchScore?: number;
+  searchable?: boolean;
+};
 
 export type PresetsMap = Map<string, Preset>;
 export type FieldsMap = Map<string, Field>;
 
 export type Metadata = {
-  projectKey?: string,
-  name?: string,
-  dataset_id?: string,
-  version?: string,
+  projectKey?: string;
+  name?: string;
+  dataset_id?: string;
+  version?: string;
 };
 
 export type Messages = {
-  [id: string]: string,
+  [id: string]: string;
 };
 
 export type State = {
   // A map of presets by preset id
-  presets: PresetsMap,
+  presets: PresetsMap;
   // A map of field definitions by id
-  fields: FieldsMap,
-  metadata: Metadata,
-  messages: Messages,
-  status: Status,
+  fields: FieldsMap;
+  metadata: Metadata;
+  messages: Messages;
+  status: Status;
 };
 
 export type ConfigContextType = [
   State,
-  { reload: () => any, replace: (fileUri: string) => any }
+  { reload: () => any; replace: (fileUri: string) => any }
```

Comment:
i'm assuming these don't actually return `any` and would be a good opportunity to change it to `void` (assuming it doesn't break a whole bunch of other stuff)